python/CCLE_data.py

```diff
@@ -11,43 +11,285 @@
 
-gene_information = mg.querymany(gene_ids, scopes='entrezgene', fields='symbol', species='human')
+#Method 1
 
+#search = search.split(" ")
 
-genes = []
+#Method 2
 
-for line in gene_information:
-    if 'symbol' not in line:
-        pass
+search = search.split(" ")
+search1 = []
+s = False
+
+for items in search:
+    word  = ""
+    for letters in items:
+        if letters == "(":
+            s = True
+        if s == True:
+            word = word + letters
+        if letters == ")":
+            s = False
+    search1.append(word)
+
+search = []
+for items in search1: 
+    items = items[1:len(items)-1]
+    search.append(items)
+        
+
+for element in search:
+    if element in gene:
+        print(element + ":", "YES")
     else:
-        gene = line['symbol']
-        genes.append(gene)
+        print(element + ":","No")
+        
+"""
+H3 RELVAL DATA
+"""
+
+os.chdir('/Users/marcdimeo/Desktop/University of Michigan Research/methylation-gem/matlab/new_var')
+
+h3_celllines = np.array(spio.loadmat('correlation_value', squeeze_me=True)["h3_ccle_names_python"])
+h3_celllines = h3_celllines.tolist()
+h3_markers =   list(dict.fromkeys(spio.loadmat('correlation_value', squeeze_me=True)["h3_marks_python"]))
+h3_expression = spio.loadmat('h3_relval', squeeze_me=True)["h3_relval"]
+
+h3_relval = pd.DataFrame(h3_expression)
+h3_relval = h3_relval.T
+h3_relval.insert(0, 'Histone', h3_markers, True)
+h3_relval.columns = ['Histone'] + h3_celllines
 
 """
-GETTING EXPRESSION DATA
+NORMALIZE H3 DATA
+"""
+
+temp_df = h3_relval.copy()
+
+del temp_df["Histone"]
+
+x = temp_df.values.astype(float)
+min_max_scaler = preprocessing.MinMaxScaler()
+x_scaled = min_max_scaler.fit_transform(x)
+h3_normalized = pd.DataFrame(x_scaled)
+
+h3_normalized.insert(0, 'Histone', h3_relval.Histone , True)
+h3_normalized.columns = ['Histone'] + h3_celllines
+
+h3_normalized = h3_normalized.fillna(method='ffill')
+
+
+
 """
+REORGANIZING DATA
+"""
+#Finding common cell lines
+h3_ccle_cellline = common_celllines(cellline_list, h3_celllines) 
+h3_ccle_cellline.sort()
+     
+ccle_h3_df = df_normalized[['Gene']+ h3_ccle_cellline]
+ccle_h3_df = ccle_h3_df.set_index('Gene')
+
+h3_ccle_df = h3_normalized[['Histone']+h3_ccle_cellline]
+h3_ccle_df= h3_ccle_df.set_index('Histone')  
+
+h3_ccle_df= h3_ccle_df.fillna(method='ffill')
+ccle_h3_df = ccle_h3_df.fillna(method='ffill')
+
+#H3 and CCLE
+h3_ccle_matrix = Matrix(search, h3_markers, ccle_h3_df, h3_ccle_df)
+    
+h3_ccle_matrix = pd.DataFrame(h3_ccle_matrix)
+h3_ccle_matrix.columns = h3_markers
+h3_ccle_matrix.insert(0, 'Genes', search , True)
+h3_ccle_matrix = h3_ccle_matrix.set_index('Genes')
+
+"""
+LEROY DATA
+"""
+os.chdir('/Users/marcdimeo/Desktop/University of Michigan Research/methylation-gem/matlab/vars')
```

Comment:
My suggestion -  make the block of code above into a function to input any matlab / text/ excel file and output the results.